Tests/sdk_release/wait_for_release.py

```diff
@@ -0,0 +1,57 @@
+import argparse
+import sys
+import time
+import requests
+import urllib3
+from Tests.scripts.utils.log_util import install_logging
+from Tests.scripts.utils import logging_wrapper as logging
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+ARTIFACTS_URL = 'https://art.code.pan.run/artifactory/api/pypi/pypi.org/simple/demisto-sdk'  # disable-secrets-detection
```

Comment:
ask Michal for the constant for `art.code.pan.run` if one doesn't exist, please create one.